sam/test-event.json

```diff
@@ -0,0 +1,123 @@
+{
+  "body": "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIFgR8x4W9X5Ix2tCx8YYB1MysVpSDj5kzAdsQJIC+iP0 paultag@nyx",
+  "resource": "/{proxy+}",
+  "path": "/path/to/resource",
+  "httpMethod": "POST",
+  "isBase64Encoded": true,
+  "queryStringParameters": {
+    "foo": "bar"
+  },
+  "multiValueQueryStringParameters": {
+    "foo": [
+      "bar"
+    ]
+  },
+  "pathParameters": {
+    "proxy": "/path/to/resource"
+  },
+  "stageVariables": {
+    "baz": "qux"
+  },
```

Comment:
Can all this stuff be removed?

Reply:
Yeah, this is the sam template. Let me try deleting everything, it may work?